site/src/components/OrganizationAutocomplete/OrganizationAutocomplete.tsx

```diff
@@ -57,11 +57,26 @@ export const OrganizationAutocomplete: FC<OrganizationAutocompleteProps> = ({
 			: [];
 	}
 
+	// Unfortunate: this useEffect sets a default org value
+	// if only one is available and is necessary as the autocomplete loads
+	// its own data. Until we refactor, proceed cautiously!
+	useEffect(() => {
+		const org = options[0];
+		if (options.length !== 1 || org === selected) {
+			return;
+		}
+
+		setSelected(org);
+		onChange(org);
+	}, [options, selected, onChange]);
+
 	return (
 		<Autocomplete
 			noOptionsText="No organizations found"
 			className={className}
 			options={options}
+			disabled={options.length === 1}
+			value={selected}
```

Comment:
Populating `value` is necessary so that the default value shows in the Autocomplete.